modules/social_features/social_album/config/install/views.view.albums.yml

```diff
@@ -946,7 +946,8 @@ display:
             enabled: '1'
       access:
         type: social_album_group
-        options: {  }
+        options:
+          group_permission: 'view group_node:topic content'
```

Comment:
@chmez Should we use album instead of topic here?

Reply:
Fixed.